cloud/app/routes/sync.py

```
@@ -0,0 +1,110 @@
+    if body.lessons:
+        rows = [
+            {
+                "brain_id": brain_id,
+                "category": l.category,
+                "description": l.description,
+                "state": l.state.value,
+                "confidence": float(l.confidence),
+                "fire_count": l.fire_count,
+                "recurrence_days": l.recurrence_days,
+            }
+            for l in body.lessons
```

Ruff (0.15.9)

[error] Ambiguous variable name: `l`

(E741)